atom/browser/native_window_mac.mm

```diff
@@ -1205,6 +1211,67 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return collectionBehavior & NSWindowCollectionBehaviorCanJoinAllSpaces;
 }
 
+void NativeWindowMac::SetVibrancy(const std::string& type) {
+  if (!(base::mac::IsOSMavericks() || base::mac::IsOSYosemiteOrLater())) return;
+
+  NSView* vibrant_view = [window_ vibrantView];
+
+  if (type.empty()) {
+    if (vibrant_view == nil) return;
+
+    [vibrant_view removeFromSuperview];
+    [window_ setVibrantView:nil];
+
+    return;
+  }
+
+  NSVisualEffectView* effect_view = (NSVisualEffectView*)vibrant_view;
+  if (effect_view == nil) {
+    effect_view = [[NSVisualEffectView alloc] initWithFrame:
```

Comment:
Last comment, does this need an `autorelease` call or will it be destroyed when `[window_ setVibrantView:nil]`?